src/check.c

```diff
@@ -149,10 +150,49 @@ TCase *tcase_create(const char *name)
     tc->ch_sflst = check_list_create();
     tc->unch_tflst = check_list_create();
     tc->ch_tflst = check_list_create();
+    tc->tags = NULL;
```

Comment:
Hi Branden,

The only reason was that the tag list is always replaced rather than being
modified so the initial list would end up being freed .. but agree that
always having a list removes a special case .. will remove.

Thx

Cris


On Fri, Jun 24, 2016 at 10:45 PM, Branden Archer <notifications@github.com>
wrote:

> In src/check.c
> <https://github.com/libcheck/check/pull/44#discussion_r68467730>:
>
> > @@ -149,10 +150,49 @@ TCase *tcase_create(const char *name)
> >      tc->ch_sflst = check_list_create();
> >      tc->unch_tflst = check_list_create();
> >      tc->ch_tflst = check_list_create();
> > +    tc->tags = NULL;
>
> The other lists are created and destroyed unconditionally, Why not follow
> this pattern? Or, is there an advantage to doing it this way?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/libcheck/check/pull/44/files/a9cd88e4e4b0864e4c416c15c78e5e06c07da514..e3f6eb3ed7044f28d835e1c063aad5d448f6be3e#r68467730>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ALhg7a6X_7ySEfYiLetqQoUuo2S2TFypks5qPFAUgaJpZM4IuPMy>
> .
>